cmd/soroban-cli/src/commands/contract/bindings/python.rs

```diff
@@ -0,0 +1,21 @@
+use std::fmt::Debug;
+
+use clap::Parser;
+use soroban_spec_json;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("generate json from file: {0}")]
+    GenerateJsonFromFile(soroban_spec_json::GenerateFromFileError),
+}
+
+impl Cmd {
+    pub fn run(&self) -> Result<(), Error> {
+        println!("Python bindings are provided by external library: https://github.com/lightsail-network/stellar-contract-bindings");
+        Ok(())
```

Comment:
Can you make this always return an error? The reason being is that the stellar-cli command whenever displaying help messaging, without the use of the `--help`, and without taking an action, returns an exit code to the user. This makes it really clear that the command hasn't done anything.

To see what I mean run the `stellar contract bindings` command and you'll see output with an error exit code (non-zero) which you can check with `echo $?` after the command.

In the error definition you can specify the text, and for the text I'd use this:
> python binding generation is not implemented in the stellar-cli, but is available via the tool located here: https://github.com/lightsail-network/stellar-contract-bindings

Note no title case is used on first word for error messages.

Reply:
a0597b360c646fa072b94fe2541a2cfcafac9237